src/Orleans/Streams/QueueAdapters/IQueueAdapterReceiver.cs

```diff
@@ -32,8 +32,6 @@ namespace Orleans.Streams
     /// </summary>
     public interface IQueueAdapterReceiver
     {
-        QueueId Id { get; }
```

Comment:
>"it is not immediately obvious why some way to identify a queue, like QueueId, is not a core functionality."

I agree, but neither is it immediately obvious why some way to identify a queue is a core functionality, which is the perquisite for including it in the interface.  We should not bloat interfaces based on the anticipation that some fields may be useful.